node/src/Pos/Client/Txp/History.hs

```diff
@@ -271,23 +272,23 @@ getBlockHistoryDefault addrs = do
 
 getLocalHistoryDefault
     :: forall ctx m. TxHistoryEnv ctx m
-    => [Address] -> m (Map TxId TxHistoryEntry)
-getLocalHistoryDefault addrs = runDBToil . evalToilTEmpty $ do
+    => MemPoolSnapshot -> [Address] -> m (Map TxId TxHistoryEntry)
+getLocalHistoryDefault mempoolSnapshot addrs = runDBToil . evalToilTEmpty $ do
```

Comment:
Similarly, if we're consistent then we can use a short name, because people cane remember.